scripts/appmenus_archlinux_minimal/whitelisted-appmenus.list

```diff
@@ -1 +0,0 @@
-xterm.desktop
```

Comment:
Those files are supposed to be here. Template builder package them together with root.img.

Reply:
Thanks :) re added them